app/templates/components/ui-table/cell/admin/groups/cell-group-name.hbs

```diff
@@ -0,0 +1,14 @@
+<span>{{t @record}}</span>
+<div class="ui hidden divider"></div>
+<div class="ui horizontal compact basic buttons">
+  <LinkTo @route="groups.edit" @model={{@extraRecords.id}}>
+    <UiPopup @content={{t "Edit"}}  @class="ui icon button">
+      <i class="edit icon"></i>
+    </UiPopup>
+  </LinkTo>
+  {{#if (not @extraRecords.deletedAt)}}
+    <UiPopup @content={{t "Delete"}} @class="ui icon button" @click={{action @props.actions.deleteGroup @extraRecords.id}}>
```

Comment:
```suggestion
<UiPopup @content={{t "Delete"}} @class="ui icon button" @click={{action (confirm (t "Are you sure you would like to delete this group?") (action @props.actions.deleteGroup @extraRecords.id))}}>
```

Reply:
we don't add confirmation in admin section. I have removed that from admin event section in past.